### DISCUSSION TITLE
sof v1.3-rc1 released - master merge window closed for one week.

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7373] [Team Posts]
See https://github.com/thesofproject/sof/releases/tag/v1.3-rc1

Only bug fixes will be merged into master, except for pending GDB patch. Expect rc2 around Thursday. Remaining known issues / PRs.
https://github.com/thesofproject/sof/milestone/2

Maintainers please don't merge new features into master.

**Comment by tlauda** [comment]
@lgirdwood What with the Keyword Detector related PRs? I was under the impression we will make release from the master, but with this 1 week freeze it's impossible.

**Comment by jajanusz** [comment]
I think wov is for 1.4, cos:
> I was thinking hot wording and IMX support for v1.4 about 8 weeks after 1.3 ? Just to let things settle on the KD side.... My hope for 1.3 was stable HDA/HDMI and PM.

Source: https://github.com/orgs/thesofproject/teams/steering-committee/discussions/2/comments/2

**Comment by lgirdwood** [comment]
@tlauda This wont stop KD patches being merged into 1.3, but gives more breathing space and less pressure since we also need userspace tools/code to accompany KD and gives it time to mature.

**Comment by lgirdwood** [comment]
Given the news that pulseaudio and BYT are broken, the merge window is back open for most feature PR's that are not invasive. RC2 out after PA and BYT fixed.
